pages/docs/plugins/grouparoo-postgres.mdx

```diff
@@ -0,0 +1,205 @@
+
+- `column`: (**required**) The name of the column to use as the high water mark. This is a date column.
+
+#### Table Source Properties
+
+After you generate a Source, you'll likely want to add Properties to it. You can do this through the CLI:
+
+    $ grouparoo generate postgres:table:property first_name --parent users
+
+You can also create properties in a batch when creating the Table Source [using the `--with` option while running the `generate` command](/docs/cli/config#batch-generation).
+
+The Property generator will drop individual files in the `config/properties` directory. Edit these files to match your desired configuration.
+
+The Property config object has several options. [Some are share across all Properties](/docs/config/properties/community#configuring-your-property), while others are more specific to the type of Property generated. A Postgres Table Source has a few unique options. These can be found in the `options` object in the config file. They are:
+
+- `column`: (**required**) The name of the column to use for the Property.
+- `aggregationMethod`: (**required**, default: `"exact"`) The type of aggregation method to use when extracting the data. The available options will be added to the generated config file as a comment.
+- `sort`: (**required**, default: `null`), // You can sort the results by another column in this table.
+
+A Postgres Table Source also provides the ability to filter your data via the `filters` option. This is a series of rules that will filter data in the database table to find the appropriate value for each Profile for a given Property.
+
+For example, let's say you had a property called `lifetime_value` which summed all the purchases for a given user. Your Source is a `purchases` table that has a `state` column set to either `successful` or `returned`. You may only want to include `successful` purchases. Your `filters` config might look like this:
+
+```js
+{
+  filters: [{ key: "state", op: "equals", match: "successful" }],
+}
+```
+
+The available operators (`op`) will be added to the generated config file as a comment near the `filters` section.
+
+### Query Source
+
+A Query Source is a more flexible way to build properties. With a Query Source, you can add custom SQL commands to your Properties, which could pull data from one or more tables in your database.
+
+The unique thing about the way a Query Source works is that there aren't any unique options for the Source itself. But the Schedule and Properties look a little different.
+
+#### Query Source Schedule
+
+Like a [Table Source Schedule](#table-source-schedule), a Query Source Schedule is included with the generated config file, commented. It has a couple unique options:
+
+- `query`: A SQL query to return that tells Grouparoo which Profiles to check each time the interval occurs.
+- `propertyId`: The `id` of the Grouparoo Property whose data is returned by `options.query`.
```

Comment:
... be sure that your query only returns on column of data, and that it maps to the Grouparoo Property you set with `options.propertyId`.